app/src/main/AndroidManifest.xml

```diff
@@ -121,7 +122,7 @@
 
         <activity
             android:uiOptions="splitActionBarWhenNarrow"
-            android:configChanges="orientation|screenSize"
+            android:configChanges="orientation|screenSize|uiMode"
```

Comment:
Why?

Reply:
To handle the theme changes whilst the app is being used (as mentioned [here](https://developer.android.com/guide/topics/ui/look-and-feel/darktheme#config-changes))

Reply:
Why don't we want an automatic restart of the activities? In which activities wouldn't we want an automatic restart?

Reply:
> Why don't we want an automatic restart of the activities? In which activities wouldn't we want an automatic restart?

 Didn't get you... We require the onConfigChanged() to get called when the change happens, hence the uiMode attribute...

Reply:
> Didn't get you... We require the onConfigChanged() to get called when the change happens, hence the uiMode attribute...

Not necessarily, you can just let the activity restart.

Reply:
Probably only some activities should update their state not restarting.

Reply:
I'm sorry, I didn't get you again. `uiMode` was added to get `onConfigurationChanged()` called when the user changes the system-level dark theme whilst using our app - so that we can apply our dark theme in the app.